### PR TITLE
Add tempo to aws collections

### PR DESCRIPTION
**Pull request by QuentinBisson**
This PR:

- adds/changes/removes etc

### Testing

Description on how tempo-app can be tested.

- [ ] fresh install works
  - [ ] AWS
  - [ ] Azure
  - [ ] KVM
- [ ] upgrade from previous version works
  - [ ] AWS
  - [ ] Azure
  - [ ] KVM

#### Other testing

Description of features to additionally test for tempo-app installations.

- [ ] check reconciliation of existing resources after upgrading
- [ ] X still works after upgrade
- [ ] Y is installed correctly

### Checklist

- [ ] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

**Comment by QuentinBisson** [comment]
Yep, I'm creating the config PR to only deploy it on gaia though :) as long as it's not released this PR can be merged :)

**Comment by QuentinBisson** [comment]
My thinking here is that it's going to be easier to just set a tracing.enabled true in one installation and have tempo there than to do some manual stuff. We can always disable it once we're done

**Comment by hervenicol** [comment]
Could you link the issue where we've decided to deploy tempo on MCs by default?

**Comment by QuentinBisson** [comment]
There is no roadmap item around tracing and if you don't want that i'll handcraft things by hand but I think it's simpler to do it right the first time.

See https://github.com/giantswarm/config/pull/2062

**Comment by hervenicol** [comment]
All right, I looked at:
- your PR in config repo that disables tracing by default - https://github.com/giantswarm/config/pull/2062/files
- the tempo app that has a condition on `tempo.enabled` to deploy the tempo dependency - https://github.com/giantswarm/tempo-app/blob/main/helm/tempo/Chart.yaml

So, it looks like when the config repo one will be merged, it will be safe to release tempo to collections.

Sorry for all the questions, it was not clear to me in your PR's description.

File: .circleci/config.yml [hunk 1]
```diff
@@ -30,3 +30,17 @@ workflows:
...
               only: /^v.*/
+
+      - architect/push-to-app-collection:
+          context: "architect"
+          name: aws-app-collection
+          app_name: "tempo"
+          app_namespace: "tempo"
+          app_collection_repo: "aws-app-collection"
+          requires:
+            - "package and push tempo chart to control-plane-catalogs"
```
Comment (hervenicol) [inline .circleci/config.yml:41]: This dependency is probably broken: I didn't find this stage in the existing `config.yml`.

**Comment by QuentinBisson** [comment]
No worries you're right about the questions and I will not merge it today :)

File: .circleci/config.yml (hunk 1 above)
Comment (QuentinBisson) [reply .circleci/config.yml]: You're right I fixed it

**Review by QuantumEnigmaa — APPROVED** [review]

**Comment by QuentinBisson** [comment]
Let's gooo

**Pull request merged**